### DISCUSSION TITLE
增加一个自动更新站点规则的功能

### DISCUSSION THREAD

**Discussion by flashlab** [ronggang/PT-Plugin-Plus#1650] [Ideas]
目前PPTP主程序已经很完善了，唯一需要频繁更新的就是站点规则，也就是`resource/sites`里的文件。
有没有可能增加一个自动更新的功能，如果dev分支有更新这部分内容，可以让插件自动获取并更新？这样也省去了每次都要从action重新下的不便。

**Comment by ted423** [comment]
很危险，这种说不定还要面临被hack的风险
